boards/renesas/ek_ra6m3/doc/index.rst

```diff
@@ -102,6 +102,8 @@ The below features are currently supported on Zephyr OS for EK-RA6M3 board:
 +-----------+------------+----------------------+
 | COUNTER   | on-chip    | counter              |
 +-----------+------------+----------------------+
+| USBHS     | on-chip    | udc                  |
++-----------+------------+----------------------+
```

Comment:
@thenguyenyf Please add test feature `usbd` to board's yaml files, example https://github.com/zephyrproject-rtos/zephyr/blob/e1b43207224e2226dc692bd294afaf869e05a201/boards/renesas/da14695_dk_usb/da14695_dk_usb.yaml#L17 In this or a subsequent PR.